languages/common-lisp/exercises/concept/sameness/.docs/instructions.md

```diff
@@ -0,0 +1,5 @@
+Leslie the Lisp Alien is programming their maze-robot (as young Lisp Aliens are wont to do) and needs some help. The maze consists of a series of rooms. Each room as a choices of doors. Each door needs a key. If you use the wrong key at a door it is likely to explode (do not worry for the robot, there will be no damage but buffing out the scorch marks is quite annoying for Leslie). If the robot doesn't have a key to open any doors then the whole room will explode! (Again, do not worry it will not damage but will be annoying to clean.)
+
+What you need to help Leslie with is to program her robot to pick the correct key for each room. In each room the robot will try the key in each door in order until it finds one that the key opens. It will then go through the door and find out what lies behind it.
+
```

Comment:
In line with above, I think we are supposed to have numbered sections here, but there is kinda only one. We could just add a header for this first (and only) task, then it will be super easy to just make all of the hints headers children of this one task header.

We might want to ask Erik or Jeremy about what to do when we just have one big section like this

Reply:
Good idea - I will tag Erik to weigh in on this issue.

Reply:
> In line with above, I think we are supposed to have numbered sections here, but there is kinda only one. We could just add a header for this first (and only) task, then it will be super easy to just make all of the hints headers children of this one task header.

Well, there is just one function to implement, but that does not mean you could not split that into several tasks :) Splitting it into several tasks has a couple of advantages:

- Students can focus on getting just one part of the functionality right, which means they'll likely feel less overwhelmed
- As hints can be linked to specific tasks, you can give more specific hints to the student. This can be especially helpful to the student

Each task should also contain an example what to return for a certain input. You can read more about that [here](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#docsinstructionsmd) or check out the [basics instructions](https://github.com/exercism/v3/blob/master/languages/common-lisp/exercises/concept/basics/.docs/instructions.md).

I like these instructions, it could just use a little tweaking in its structure and then we're good to go!